starter-code/index.html

```diff
@@ -10,15 +10,19 @@
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
     <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <script id="blog-articles" type="text/x-handlebars-template">
+      <article>
```

Comment:
One last thing. The filters don't work. 
Maybe you want to add something here to make it work?